admin/taxonomy/class-taxonomy-columns.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+/**
+ * This class adds columns to the taxonomy table.
+ */
+class WPSEO_Taxonomy_Columns {
+
+	/**
+	 * WPSEO_Taxonomy_Columns constructor.
+	 */
+	public function __construct() {
+
+		$this->taxonomy = FILTER_INPUT( INPUT_GET, 'taxonomy' );
+
+		add_filter( 'manage_edit-' . $this->taxonomy . '_columns', array( $this, 'add_columns' ) );
```

Comment:
It might be better to wrap these filters in a `! empty( $this->taxonomy )` conditional?